backend/src/tasks/dnstwist.ts

```diff
@@ -29,16 +29,25 @@ export const handler = async (commandOptions: CommandOptions) => {
   await connectToDatabase();
   const dateNow = new Date(Date.now());
   console.log('Running dnstwist on organization', organizationName);
-  const domainsWithIPs = await getIps(organizationId);
+  const root_domains = await getRootDomains(organizationId!);
   const vulns: Vulnerability[] = [];
-  for (const domain of domainsWithIPs) {
+  console.log(root_domains);
+  for (const root_domain of root_domains) {
     try {
-      const results = await runDNSTwist(domain);
+      const results = await runDNSTwist(root_domain);
+
+      // Fetch existing domain object
+      const domain = await Domain.findOne({
```

Comment:
What happens if no Domain is found for a particular root domain? Can you add a test to cover this case (so we can ensure it doesn't crash)?

Reply:
@aloftus23 oh wait, I was thinking it would be simpler to just skip running dnstwist if the root domain doesn't exist (no need to actually create a new root domain -- doesn't the `rootDomainSync` scan already do that?) Mainly I just wanted the scan to not crash in this scenario.

Reply:
Yeah, rootDomainSync does that, just thought we'd take care of adding it during the run. Either works for me.